tsconfig.json

```diff
@@ -4,8 +4,7 @@
     "useDefineForClassFields": true,
     "lib": ["DOM", "DOM.Iterable", "ESNext"],
     "allowJs": true,
-    "skipLibCheck": false,
-    "esModuleInterop": true,
+    "skipLibCheck": true,
```

Comment:
this seemed to fix some build stuff, removing esmodule interop was optional i guess in searchv-3